courses/tests.py

```
@@ -1,3 +1,84 @@
+import os
+import django
+
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "project.settings")
+django.setup()
+
 from django.test import TestCase
```

Ruff (0.8.2)

Module level import not at top of file

(E402)